spec/rubocop/cop/rspec/empty_line_after_final_let_spec.rb

```diff
@@ -136,6 +160,29 @@
   end
   RUBY
 
+  include_examples 'autocorrect',
+                   bad_example,
+                   good_example
+
+  bad_example = <<-RUBY
+  RSpec.describe User do
+    let(:params) { foo }
+    # end of setup
```

Comment:
Since the code implements checks for multi-line comments, shouldn’t we let one of the specs have multi-line comments?